python/dalex/dalex/_arena/plots/_accumulated_dependence_container.py

```diff
@@ -0,0 +1,40 @@
+from pandas.api.types import is_numeric_dtype
```

Comment:
minor: I believe `import pandas as pd`, and then `pd.api.types.is_numeric_dtype` in code, makes it easier to read